### ISSUE TITLE
QuACS should list the waitlist capacity/how many people are on the waitlist for a section

### ISSUE THREAD

**Issue by benjaminrsherman** [quacs/quacs#25]

**Comment by elihschiff** [comment]
I dont think we should do this as we dont have data that is live for this info. The only way to get this info is to scrape sis and I dont really want to scrape sis more than once a day, and I think that showing info like this that could be 1 day old is not worth it. 

Maybe we could add a link to sections where you can click and view it on sis? For example https://sis.rpi.edu/rss/bwckschd.p_disp_detail_sched?term_in=202009&crn_in=25565

**Comment by benjaminrsherman** [comment]
That works.  If we put it in the section info modal then it won't be too obtrusive.

**Comment by elihschiff** [comment]
Sure we can add the size of the waitlist, along with the link.

Maybe, I do not know if this is technically possible, we could also add a button in the modal that when clicked will scrape the page just for that class and bring in an updated wait list size
